components/scripts/lib/build-scan-parse.sh

```diff
@@ -112,6 +120,10 @@ parse_build_scan_row() {
       build_outcomes[run_num]="${field_9}"
     fi
 
+    # The below fields are always set by Build Scan data regardless of their
+    # previous value and are always safe to override.
+
+    build_scan_ids[run_num]="${field_4}"
```

Comment:
Even though this is field 4, I thought it made sense to move it down here so that all fields are grouped according to how they are set. Let me know what you think.

Reply:
I think what you are saying is that if we already know what the Build Scan ID is then there is no reason to override it?

That is actually the same way we treat `base_urls` and `build_scan_urls` so actually I think it can move to be right next to those.

Reply:
LGTM.